Find a functionally equivalent code:android.view.ViewRootImpl.InputStage.onDeliverToNext:COMMENT
Method Modifier: protected   
Comment:/**
 * Called when an event is being delivered to the next stage.
 */

@@ -1,4 +1,7 @@
 {
+    if (DEBUG_INPUT_STAGES) {
+        Log.v(TAG, "Done with " + getClass().getSimpleName() + ". " + q);
+    }
     if (mNext != null) {
         mNext.deliver(q);
     } else {

